src/qvimagecore.cpp

```diff
@@ -68,7 +68,7 @@ QVImageCore::QVImageCore(QObject *parent) : QObject(parent)
     settingsUpdated();
 }
 
-void QVImageCore::loadFile(const QString &fileName)
+void QVImageCore::loadFile(const QString &fileName, bool isReloading)
```

Comment:
The cache key includes the file size, but this ensures it handles situations in which the size doesn't change (e.g. an uncompressed BMP was re-saved).